setup.py

```diff
@@ -4,19 +4,21 @@
 """The setup script."""
 
 from setuptools import setup, find_namespace_packages
+from pip._internal.req import parse_requirements
+from pip._internal.download import PipSession
```

Comment:
If possible, I'd avoid importing private stuff from other packages. Implementation can change with any new version without warning, thus breaking our code.

Reply:
I agree. Incidentally, this private API broke with pip 20.